lib/devise/failure_app.rb

```diff
@@ -103,11 +103,11 @@ def i18n_message(default = nil)
         options[:scope] = "devise.failure"
         options[:default] = [message]
         auth_keys = scope_class.authentication_keys
-        keys = (auth_keys.respond_to?(:keys) ? auth_keys.keys : auth_keys).map { |key| scope_class.human_attribute_name(key) }
+        keys = (auth_keys.respond_to?(:keys) ? auth_keys.keys : auth_keys).map { |key| scope_class.human_attribute_name(key).downcase }
```

Comment:
Could we add tests for it? 🤔

Reply:
Edited the existing tests. They check, that the downsizing are done. There's not really a test to check the added humanize method though, not sure, if that's necessary?

Reply:
I would add to ensure that it's the expected behavior.